hyperspy/_components/gaussian.py

```diff
@@ -70,7 +70,7 @@ class Gaussian(Expression):
     ============== ===========
     Variable        Parameter
     ============== ===========
-    :math:`A`       A
```

Comment:
As I understand it, this `A` is refering to the parameter name, as in the name of the hyperspy parameter object. I think, in this case, it makes sense to keep `A`, so that user can easily find the parameter names from the docstring.
The meaning of `A` is then explain below in the parameter docstring.